game/map.py

```diff
@@ -0,0 +1,8 @@
+class Maproom:
+    Worldmap = []
```

Comment:
Na chvili jeste muzem zapomenout na mapu sveta, jde nam prvni o to, mit seznam hracu a priser na jednom miste... zaloz teda attributy monsters a players, coz budou na zacatku prazdne listy